applications/crossbar/doc/token_restrictions.md

```diff
@@ -277,64 +318,40 @@ If endpoint matching fails to find a match, Crossbar will try to match the HTTP
 
 List can contain any valid HTTP method ("GET", "PUT", "POST", "PATCH", "DELETE") or the "catch all" - `"_"`.
 
-#### Manage accounts templates
+#### Schema
 
-##### Get current account template
+Key | Description | Type | Default | Required
+--- | ----------- | ---- | ------- | --------
+`restrictions` |   | `object` |   | `false`
 
-`GET /v2/accounts/{ACCOUNT_ID}/token_restrictions` return JSON like this:
 
-```JSON
-{
-  "data": {
-    "restrictions": {
-      "{AUTH_METHOD}": {
-        "{PRIV_LEVEL}": {
-          "{API_ENDPOINT}": [
-            {
-              "allowed_accounts": {ACCOUNT_IDS},
-              "rules": {
-                "{ARGUMENT_MATCH}":{HTTP_METHODS}
-              }
-            }
-          ]
-        }
-      }
-    }
-  }
-}
-```
+#### Remove
 
-If the account doesn't have token restrictions, the API will return a 404 error.
+> DELETE /v2/accounts/{ACCOUNT_ID}/token_restrictions
 
-##### Create/update account template
+```curl
+curl -v -X DELETE \
+    -H "X-Auth-Token: {AUTH_TOKEN}" \
+    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/token_restrictions
+```
 
-`POST /v2/accounts/{ACCOUNT_ID}/token_restrictions`
+#### Fetch
```

Comment:
Can you make the titles more descriptive? I auto-generate the stub based on the HTTP method, but expand the title when modifying the doc. So, this could be `Fetch the account's configured token restrictions` or similar.